purchase_order_product_recommendation/wizards/purchase_order_recommendation.py

```diff
@@ -42,10 +42,20 @@ class PurchaseOrderRecommendation(models.TransientModel):
              'Leave it as 0 to set no limit',
     )
     show_all_partner_products = fields.Boolean(
-        string='Show all products',
+        string='Show all supplier products',
         default=False,
         help='Show all products with supplier infos for this supplier',
```

Comment:
```suggestion
         help='Show all products with supplier info for this supplier',
```

Reply:
Out of the aim of this PR, If product variant is not match at vendor price, it should display product template price, shouldn't it?

Reply:
Mmm. I don't think so. It should be 0, as there's no price history with that provider yet and you don't have much control on it anyway.